packages/Popover/src/components/Trigger/Trigger.js

```diff
@@ -50,17 +50,22 @@ function Trigger(props) {
   const { children, a11yText, ...moreProps } = props;
 
   if (typeof children === "function") {
-    const a11yAttributes = { "aria-describedby": content.ariaId };
+    const a11yAttributes = { "aria-describedby": content.ariaId, "aria-haspopup": true };
     return React.cloneElement(children(handleTriggerEvent, a11yAttributes, isOpen), {
       ...moreProps,
     });
   }
 
+  const commonAttributes = {
+    a11yText,
+    "aria-haspopup": true,
+    "data-pka-anchor": "popover.trigger",
+  };
+
   if (isEager) {
     return (
       <sc.Trigger
-        a11yText={a11yText}
-        data-pka-anchor="popover.trigger"
+        {...commonAttributes}
```

Comment:
`isEager ` is always for tooltips right?  but is tooltip a popup?

Reply:
Nope.  A tooltip is not a popup, it's a different pattern.
https://www.w3.org/TR/wai-aria-practices-1.1/#tooltip

I think we actually follow that pattern pretty well, though probably don't add a `role=tooltip` on the `<Popover.Content>` when `isEager=true`.

Reply:
Seems like the logic does need to be a little more sophisticated.  There's four cases I can see: 
  - tooltip render prop
  - popover render prop
  - tooltip trigger
  - popover trigger
 
The tooltip version should have `aria-describedby=content.ariaId`.
The popover version should have the `aria-haspopup` and `aria-controls=content.ariaId`.